docs/source/1.0/spec/core/model.rst

```diff
@@ -828,10 +828,9 @@ The following example defines a union shape with several members:
         union MyUnion {
             i32: Integer,
 
-            stringA: String,
+            string: String,
 
-            @sensitive
```

Comment:
This was to show that traits can also be applied to members of unions, can one be added here?